docs/design/autoware-concepts/difference-from-ai-and-auto.md

```diff
@@ -0,0 +1,37 @@
+# How is Autoware Core/Universe different from Autoware.AI and Autoware.Auto?
+
+Autoware is the world's first "all-in-one" open-source software for self-driving vehicles.
+Since it was first released in 2015, there have been multiple releases made with differing underlying concepts, each one aimed at improving the software.
+
+## Autoware.AI
+
+[Autoware.AI](https://github.com/Autoware-AI/autoware.ai) is the first distribution of Autoware that was released based on ROS 1. The repository contains different packages to cover different aspects of autonomous driving technologies, including sensing, actuation, localization, mapping, perception, planning, and so on.
+
+While it was successful in attracting many developers and contributions, there were no particular use-cases nor an ODD defined for the software. It also lacked design documents to specify architecture, and each package had different coding standards with very low test coverage.
+
+From the lessons learned from Autoware.AI development, a different development process was taken for Autoware.Auto to develop a ROS 2 version of Autoware.
+
+!!! warning
+Autoware.AI is currently in maintenance mode and will reach end-of-life at the end of 2022.
+
+## Autoware.Auto
+
+[Autoware.Auto](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto) is an autonomous driving software based on ROS 2. As part of the transition to ROS 2, it was decided to avoid simply porting Autoware.AI from ROS 1 to ROS 2. Instead, the codebase was rewritten from scratch with proper engineering practices, including defining target use cases and ODDs (eg: Autonomous Valet Parking [AVP], Cargo Delivery, etc.), designing a proper architecture, writing design documents and test code.
```

Comment:
[IMO]
- `an autonomous driving software` is obvious and redundant.
- Using the same format as Autoware.AI looks nice?

```suggestion
[Autoware.Auto](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto) is the second distribution of Autoware that was released based on ROS 2. As part of the transition to ROS 2, it was decided to avoid simply porting Autoware.AI from ROS 1 to ROS 2. Instead, the codebase was rewritten from scratch with proper engineering practices, including defining target use cases and ODDs (eg: Autonomous Valet Parking [AVP], Cargo Delivery, etc.), designing a proper architecture, writing design documents and test code.
```